File: works/tmp/2577.py
a = int(input())
b = int(input())
c = int(input())
res = a * b * c
res_s = str(res)

mark = [0 for _ in range(10)]
for i in range(len(res_s)):
    mark[int(res_s[i])] += 1
    
for i in range(10):
    print(mark[i])

File: works/js/4-1/2577.js
let fs = require('fs');
let input = fs.readFileSync(0).toString().trim().split('\n')
// let input = fs.readFileSync('input.txt').toString().split('\n')

let a = parseInt(input[0]), b = parseInt(input[1]), c = parseInt(input[2])
let res = a * b * c 
let res_s = res.toString()
let mark = Array(10).fill(0)
let answer = ""

for(i = 0; i < res_s.length; i++) {
    mark[parseInt(res_s[i])]++ 
}

for(i = 0; i < 10; i++) {
    answer += mark[i] + '\n'
}

console.log(answer) 